apps/docs/src/guide/contracts/deploying-contracts.md

```diff
@@ -37,4 +37,4 @@ Now that the contract is deployed, you can interact with it. In the following st
 
 <<< @/../../docs-snippets/src/guide/contracts/deploying-contracts.test.ts#contract-setup-4{ts:line-numbers}
 
-For a more comprehensive TypeScript-backed Fuel usage, learn how to [generate types from ABI](../typegen/generating-types.md)
+For a more comprehensive TypeScript-backed Fuel usage, learn how to [generate types from ABI](../fuels-cli/generating-types.md)
```

Comment:
This may not be directly related to this PR but what does _TypeScript-backed_ mean? Is it that the generated types are Typescript types?

Reply:
Yes exactly, but if you think that isn't clear it's probably worth rewording this at some point.